kolibri/core/assets/src/state/actions.js

```diff
@@ -428,39 +428,53 @@ function saveLogs(store) {
   }
 }
 
+function fetchPoints(store) {
+  if (!getters.isSuperuser(store.state) && getters.isUserLoggedIn(store.state)) {
```

Comment:
I also cleaned up the `getters` include statement in #1792 , which should make this a little easier to read once I merge this in.